api/src/models/website.model.js

```diff
@@ -3,6 +3,8 @@
  */
 const mongoose = require('mongoose');
 
+const { layoutOptionEnum, sortingOptionEnum } = require('../config/publication');
```

Comment:
Oh I see why you renamed that playwright file now. Is it to encapsulate those different enum constants? In that case, perhaps create a new file specifically meant to house publication constants? It's just that I don't want to mix config related to playright with config related to our schema.

Reply:
Or, just inline their definitions here if those constants are not used anywhere else? 
I'd say follow what the rest of the model files are doing. If they've hardcoded enum values in the schema definition, it'd be better to just follow that style to maintain consistency.